src/main/resources/edu/wpi/teamb/views/PatientTransportation.fxml

```diff
@@ -9,192 +9,326 @@
+                           <font>
+                              <Font size="36.0" />
+                           </font>
+                           <VBox.margin>
+                              <Insets left="40.0" top="40.0" />
+                           </VBox.margin>
+                        </Text>
+                        <HBox layoutX="50.0" layoutY="170.0">
                            <children>
-                              <TextField fx:id="employeeIDField" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" onKeyReleased="#buttonControl" prefHeight="30.0" prefWidth="130.0" />
-                              <TextField fx:id="emailField" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" onKeyReleased="#buttonControl" prefHeight="30.0" prefWidth="130.0" />
+                              <Text strokeType="OUTSIDE" strokeWidth="0.0" text="Urgency">
+                                 <font>
+                                    <Font size="22.0" />
+                                 </font>
+                                 <HBox.margin>
+                                    <Insets left="40.0" />
+                                 </HBox.margin>
+                              </Text>
+                              <Text layoutX="10.0" layoutY="34.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Assigned Staff Member">
+                                 <font>
+                                    <Font size="22.0" />
+                                 </font>
+                                 <HBox.margin>
+                                    <Insets left="299.0" />
+                                 </HBox.margin>
+                              </Text>
                            </children>
-                           <padding>
-                              <Insets bottom="15.0" right="235.0" />
-                           </padding>
                            <VBox.margin>
-                              <Insets />
+                              <Insets top="10.0" />
                            </VBox.margin>
                         </HBox>
-                        <Text strokeType="OUTSIDE" strokeWidth="0.0" text="Request Information">
-                           <font>
-                              <Font size="24.0" />
-                           </font>
+                        <HBox layoutX="10.0" layoutY="199.0" prefHeight="22.0" prefWidth="200.0">
+                           <children>
+                              <ChoiceBox fx:id="urgencyBox" onAction="#buttonControl" prefHeight="44.0" prefWidth="340.0">
+                                 <HBox.margin>
+                                    <Insets left="40.0" />
+                                 </HBox.margin>
+                              </ChoiceBox>
+                              <TextField fx:id="assignedEmployeeField" minHeight="44.0" onKeyReleased="#buttonControl" prefWidth="340.0">
```

Comment:
need to implement the assigned employee field in the controller so it gets cleared when the clear button is clicked. Currently not working. See how I did it for Sanitation controller